md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/SubProcessAPI.java

```diff
@@ -99,7 +101,7 @@ public RestWrapper get(
             LOGGER.info("Record with ID:" + processId + " selected from Process by User:" + principal.getName());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Record with ID:" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)